react/components/MinicartFreeshipping.tsx

```diff
@@ -9,9 +10,17 @@ import styles from './MinicartFreeshipping.css'
 import AppSettings from './minicartbarSettings.graphql'
 
 interface SettingsProps {
-  settings: Settings
+  settings: BindingBoundedSettings
 }
+
+interface BindingBoundedSettings extends Settings {
+  bindingBounded?: boolean
+  settings?: [Settings]
+  freeShippingAmount: number
```

Comment:
I think that since we are extending `Settings` there's no need to have `freeShippingAmount` here.

Reply:
Ah yes, you're right. Removed that one from the interface. Cheers